src/features/dead-code/deadCodeTreeProvider.ts

```
@@ -0,0 +1,150 @@
+import * as vscode from 'vscode';
+import * as path from 'path';
+import { DeadCodeItem, DeadCodeReport } from './deadCodeAnalyzer';
+
+export class DeadCodeTreeProvider implements vscode.TreeDataProvider<DeadCodeTreeItem> {
+    private _onDidChangeTreeData: vscode.EventEmitter<DeadCodeTreeItem | undefined | null | void> = 
```

Biome (1.9.4)

[error] void is confusing inside a union type.

Unsafe fix: Use undefined instead.


(lint/suspicious/noConfusingVoidType)